README.md

```diff
@@ -1,3 +1,54 @@
-# service-center [![Build Status](https://travis-ci.org/ServiceComb/service-center.svg?branch=master)](https://travis-ci.org/ServiceComb/service-center)
-A standalone service center to allow services to register their instance information and to discover providers of a given service
-EDIT1
+# Service Center [![Build Status](https://travis-ci.org/ServiceComb/service-center.svg?branch=master)](https://travis-ci.org/ServiceComb/service-center)
+
+A standalone Service Center to allow services to register their instance information and to discover providers of a given service
+
+- [http://www.servicecomb.io/service-center](http://www.servicecomb.io/service-center)
+
+## Quick Start
+
+### Getting Service Center
+
+The easiest way to get Service Center is to use one of the pre-built release binaries which are available for OSX, Linux, Windows, and Docker. Instructions for using these binaries are on the [GitHub releases page][github-release].
+
+[github-release]: https://github.com/servicecomb/service-center/releases/
+
+### Running Service Center
+
+First, you need to run a etcd(version: 1.3.x) as a database service，and then modify the etcd IP and port in the Service Center configuration file (./conf/app.conf : manager_cluster).
```

Comment:
this statement `First, you need to run a etcd(version: 1.3.x) as a database service` probably belongs to the design of service center. 

It's better to put this information to a design section. users just need to know how to start service center in this section.